sdk/appconfiguration/ci.yml

```diff
@@ -14,7 +18,6 @@ pr:
     include:
     - sdk/appconfiguration/
     - sdk/core/
-    - eng/
```

Comment:
Is this really worth it? I feel sdk/core changes will have the same pain.

Reply:
As discussed, going to remove /sdk/core too.